### ISSUE TITLE
Gradient Accumulation Adjustment on Loss and Learning Rate 

### ISSUE THREAD

**Issue by somuchtome** [modelscope/ms-swift#2527]
https://github.com/modelscope/ms-swift/blob/d2b45ec8c6acbfc126f784f3781aac31db1c297a/swift/aigc/animatediff.py#L383

Hi, thank you very much for your documentation! I have a question regarding training with animatediff in the code: the loss is not divided by the gradient accumulation, and the learning rate is also not multiplied by the gradient accumulation. Would this have an impact on the training results?
